Fixtures/Miscellaneous/EchoExecutable/Sources/secho/main.swift

```diff
@@ -2,6 +2,8 @@
 	import Glibc
 #elseif canImport(Musl)
 	import Musl
+#elseif canImport(Android)
+	import Android
```

Comment:
Likewise.

Reply:
```
Miscellaneous_EchoExecutable/Sources/secho/main.swift:11:11: error: cannot find 'getcwd' in scope
     9 | #endif
    10 |
    11 | let cwd = getcwd(nil, Int(PATH_MAX))
       |           `- error: cannot find 'getcwd' in scope
    12 | defer { free(cwd) }
    13 | let workingDirectory = String(validatingUTF8: cwd!)!
```
`getcwd()` is declared in `unistd.h`, [which is in the Android overlay](https://github.com/swiftlang/swift/blob/22817ccf1a13188a81d42d5fa56cbac28c46adc4/stdlib/public/Platform/SwiftAndroidNDK.h#L98) but [not in the Bionic module that Swift uses](https://github.com/swiftlang/swift/blob/98b74d801021ac158c1161778ec356930438e343/stdlib/public/Platform/SwiftBionic.h).

The rest may work as `import Bionic` though, I will do more testing and let you know.